.github/workflows/stale.yml

```
@@ -0,0 +1,22 @@
+name: Close stale PRs
+on:
+  schedule:
+    - cron: "0 0 * * *" # Runs every day at midnight
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - uses: actions/stale@v9
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          days-before-pr-stale: 180 # 6 months before warning
+          days-before-pr-close: 185 # Another 6 months until closing
+          stale-pr-label: "stale"
+          stale-pr-message: "This PR has been inactive for 6 months. If it's still relevant, please update it to keep it open."
```

Let's give it a friendly tone to encourage updates :)

```suggestion
          stale-pr-message: "Hi! This PR has been inactive for 6 months. If it's still relevant, please update it to let us know you’d like to keep it open! 😊"
```